worlds/sc2/client.py

```diff
@@ -707,6 +714,8 @@ def on_package(self, cmd: str, args: dict) -> None:
                     (" to install.")
                 ).send(self)
                 self.data_out_of_date = True
+            
+            ColouredMessage("[b]Check the Launcher tab to start playing.[/b]").send(self)
```

Comment:
(super minor, optional) I wonder if we want to colour-code tabs/UI elements. Like set "Launcher tab" to show up the same orange as commands?